### DISCUSSION TITLE
[#995489] Gizmo disappearing when pressing f1 in crosshair mode

### DISCUSSION THREAD

**Discussion by King-mine** [Mojang/minecraft-editor#72] [Bugs]
The gizmo (white box and arrows) should be shown but it isn't, not sure if it is intended

Repro steps
1. Go to Crosshair mode
2. Press f1
3. Go back to tool mode
4. Select a box using the selection tool

Version: 1.19.80.20

<img width="248" alt="image" src="https://user-images.githubusercontent.com/102407943/224479876-a282eced-948a-4e97-a3ce-213a5fc0352c.png">

**Comment by minecraftchloe** [comment]
Wow! GREAT FIND!!

**Comment by minecraftchloe** [comment]
Resolved!
